net_util/src/queue_pair.rs

```diff
@@ -92,8 +92,12 @@ impl TxVirtio {
                 };
                 if result < 0 {
                     let e = std::io::Error::last_os_error();
-                    error!("net: tx: failed writing to tap: {}", e);
                     queue.go_to_previous_position();
+                    if e.kind() == std::io::ErrorKind::WouldBlock {
```

Comment:
We used `libc::EAGAIN` in the RX version, don't you think we should keep it in sync? Either way works for me (if you want to update the RX code).

Reply:
Yeh, I switched both to use `ErrorKind::WouldBlock`